jumpscale/sals/marketplace/apps_chatflow.py

```diff
@@ -39,6 +39,7 @@ def _init_solution(self):
         self.username = self.user_info()["username"]
         self.solution_metadata["owner"] = self.username
         self.threebot_name = j.data.text.removesuffix(self.username, ".3bot")
+        self.ip_version = None
```

Comment:
we can override at the start of any shared step, no?

Reply:
done, thanks